Automattic/wp-calypso
# Redesing BlazePress Campaigns for mobiles

Related to https://jira.tumblr.net/browse/SELFSERVE-506

## Proposed Changes

* Address stats data structure provided by the campaigns endpoint in Campaign type
* Make Campaign Tab table columns responsive
* Add mobile version for Campaigns Tab

## Testing Instructions



* Build assets (`yarn && yarn start`)
* Make sure your local DSP has the changes from the PR (https://github.tumblr.net/Tumblr/a8c-dsp/pull/166)
* Ensure you have the changes mentioned [here](https://github.tumblr.net/Tumblr/a8c-dsp/pull/166#issuecomment-621238) (to add stats data to campaign response)
* Mock data in the `addCampaignStatsData` function in DSP to look like (because the data should be got from Druid and it probably doesn't have data for your campaign):
```js
campaignStats.impressions_total = dailyStats?.impressions || 9332;
campaignStats.clicks_total = dailyStats?.clicks || 111;
```
* Try different screen sizes from 320px and beyond

How it might work:

https://github.com/Automattic/wp-calypso/assets/115007291/e9e1fd54-8ff8-4ea5-b02b-77a282b46a2c

## Pre-merge Checklist
